src/Components/NavBar/helpers.ts

```diff
@@ -22,3 +27,23 @@ export const updateConversationCache = (conversationCount?: number) => {
       : cookie.set("conversation-count", conversationCount)
   }
 }
+
+export const checkAndSyncIndicatorCounts = (counts: Counts) => {
```

Comment:
I will be glad to have any suggestions on the name. maybe `checkAndSyncIndicators` would be better?